### PR TITLE
[fabric] Update ansible playbooks and roles

### LINKED ISSUE #2556: update(fabric): ansible playbooks and roles to use the new standalone helm charts
### Description
As a developer I want to cleanup, update and test ansible playbooks and roles so that the Hyperledger Fabric network can be deployed using the new standalone helm charts

### Acceptance Criteria
1. Network deployed using ansible playbook (with Proxy/No Proxy and Vault)
2. Network cleanup using ansible playbook

### PR DESCRIPTION
**Pull request by mgCepeda**
**Primary Changes**

1.Deploy fabric 2.5.4 without channel using ansible. 
2.Deploy fabric 2.2.2 without channel using ansible.
3.Update deploy-network.yalm
4.Update cleanup.yaml
5. Updated samples.
6. Updated documentation docs/source/guides/networkyaml-fabric.md

fixes #2556

### CODE REVIEW

File: platforms/shared/configuration/roles/create/job_component/templates/value_peer.tpl [hunk 1]
```diff
@@ -0,0 +1,143 @@
+
+global:
```
Comment (sownak) [inline platforms/shared/configuration/roles/create/job_component/templates/value_peer.tpl:2]: Peer should also be a HelmRelease

**Review by sownak — CHANGES_REQUESTED** [review]

File: platforms/hyperledger-fabric/charts/fabric-catools/values.yaml [hunk 2]
```diff
@@ -69,12 +69,24 @@ orgData:
...
+  #     identity: user2
+  #     attributes:
+  #     - key: "hf.Revoker"
+  #       value: "true"
   usersList:
-    - identity: user1
-      attributes:
-      - key: "hf.Revoker"
-        value: "true"
+    # - identity: user1
```
[the remark this answers is not in the document]
Comment (mgCepeda) [reply platforms/hyperledger-fabric/charts/fabric-catools/values.yaml:86]: Done

File: platforms/hyperledger-fabric/charts/fabric-peernode/values.yaml [hunk 3]
```diff
@@ -81,12 +81,24 @@ certs:
...
-        - key: "hf.Revoker"
-          value: "true"
+      # - user:
+      #   identity: user1
+      #   attributes:
+      #   - key: "hf.Revoker"
+      #     value: "true"
     #Base64 encoded list of users
     #Eg. IC0gdXNlcjoKICAgICAgICAgIGlkZW50aXR5OiB1c2VyMQogICAgICAgICAgYXR0cmlidXRlczoKICAgICAgICAgICAgLSBrZXk6IGtleTEKICAgICAgIgICAgICAgIC0ga2V5OiBrZXkyCiAgICAgICAgICAgICAgdmFsdWU6IHZhbHVlMgogICAgICAgIC0gdXNlcjoKICAgICAgICAgIGlkZW50aXR5OiB1c2VyMgogICAgICAgICAgYXR0cmlidXRlczoKICAgICAgICAgICAgLSBrZXk6IGtleTEKICAgICAgICAgICAgICB2YWx1ZTogdmFsdWUxCiAgICAgICAgICAgIC0ga2V5OiBrZXkzCiAgICAgICAgICAgICAgdmFsdWU6IHZhbHVlMw==
     usersListAnsible:
```
[the remark this answers is not in the document]
Comment (mgCepeda) [reply platforms/hyperledger-fabric/charts/fabric-peernode/values.yaml:104]: Done

File: platforms/shared/configuration/roles/create/job_component/templates/value_peer.tpl (hunk 1 above)
Comment (mgCepeda) [reply platforms/shared/configuration/roles/create/job_component/templates/value_peer.tpl:2]: Done

File: platforms/shared/configuration/roles/create/job_component/templates/ca-server.tpl [hunk 4]
```diff
@@ -0,0 +1,68 @@
+global:
```
[the remark this answers is not in the document]
Comment (mgCepeda) [reply platforms/shared/configuration/roles/create/job_component/templates/ca-server.tpl:1]: Done

**Review by sownak — APPROVED** [review]

**Comment by mgCepeda** [comment]
Updated core.yaml localization

File: platforms/hyperledger-fabric/charts/fabric-peernode/templates/configmap.yaml [hunk 5]
```diff
@@ -103,24 +101,4 @@ metadata:
     app.kubernetes.io/release: {{ .Release.Name }}
     app.kubernetes.io/managed-by: helm
 data:
-  core.yaml: {{ .Files.Get "files/core.yaml" | nindent 8 | quote }}
-{{ end }}
-{{- $orderercrt := .Files.Get "files/orderer.crt" }}
```
Comment (sownak) [inline platforms/hyperledger-fabric/charts/fabric-peernode/templates/configmap.yaml:108]: orderercrt should be created, it is needed for peer cli and chaincodes

Comment (mgCepeda) [reply platforms/hyperledger-fabric/charts/fabric-peernode/templates/configmap.yaml:108]: Done

**Review by sownak — APPROVED** [review]

**Pull request merged**